content/practices/color-settings/color-settings-practice.html

```diff
@@ -0,0 +1,1197 @@
+                  <li><a href="https://support.google.com/android/answer/9730472?hl=en">Android</a></li>
+                  <li><a href="https://help.gnome.org/users/gnome-help/stable/display-night-light.html.en">Linux/GNOME</a></li>
+                </ul>
+              </td>
+              <td>
+                An operating system setting that enables users to choose between a <code>light</code> and <code>dark</code> color scheme.
+                The default is usually the <code>light</code> color scheme.
+                Typically, this setting does not automatically change rendering of native apps or web content, so it only works when developers provide support for it.
+                Many people switch to the <code>dark</code> color scheme in dark settings, e.g. at night, because it can make content easier to read or be less disruptive to other people.
+                However, people with certain visual disabilities prefer a dark color scheme as their default.
+              </td>
+              <td>
+                The CSS media query <code>prefers-color-scheme</code> identifies the current color scheme by returning <code>light</code> or <code>dark</code>.
+                Develop styles for dark text on a light background and light text on a dark background and use this media query to apply the appropriate style.
+                Ensure the text content and components for both settings meet
+                <a href="https://www.w3.org/TR/WCAG21/#contrast-minimum">WCAG 1.4.3: Contrast (Minimum)</a>.
+              </td>
+            </tr>
+            <tr>
+              <th>
+                Contrast Themes<br>
+                (AKA Forced Colors)
+              </th>
+              <td><a href="https://support.microsoft.com/en-us/windows/change-color-contrast-in-windows-fedc744c-90ac-69df-aed5-c8a90125e696">Windows 10 and later.</a></td>
+              <td>
+                Operating system accessibility feature that automatically forces all native apps and web content to render using an alternative color theme chosen by the user.
+                Some of the themes are designed to provide extra high contrast, and users can customize their chosen color theme.
+                Browsers replace author specified colors with user specified colors based on the type of HTML element.
+                Browser selections for replacement colors generally respect the system colors described below.
+                Note that ARIA attributes do not inform replacement color selection, e.g., a <code>div</code> with role <code>button</code> renders with text colors, not button colors.
+              </td>
+              <td>
+                When a user selects a contrast theme, the CSS media query <code>forced-colors</code> is set to <code>active</code>.
+                Use <code>currentcolor</code> and <code>system-color</code> values to style user interface controls and custom widgets to the user preferences.
```

Comment:
@jongund  given what you have been saying about currentcolor, I am assuming you want to change this line? Maybe just delete the part about currentcolor?